Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -186,6 +346,61 @@ final public class LottieAnimationView: LottieAnimationViewBase {
   /// Value Providers that have been registered using `setValueProvider(_:keypath:)`
   public private(set) var valueProviders = [AnimationKeypath: AnyValueProvider]()
 
+  /// Sets the current animation time with a Progress Time
+  ///
+  /// Note: Setting this will stop the current animation, if any.
+  /// Note 2: If `animation` is nil, setting this will fallback to 0
+  public var currentProgress: AnimationProgressTime {
```

Comment:
Can you revert the changes to these properties? Otherwise LGTM

Reply:
you want me to make `currentProgress`/`currentFrame`/`currentTime` open again? Got the change to make them public in very fast before you commented this

Reply:
Sorry, I mean make it so those properties aren't included this PR / the diff. It looks like they were moved to a different place in the file.